saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/classifier/ClassifierUtils.scala

```diff
@@ -165,6 +167,14 @@ object ClassifierUtils extends Logging {
     }
   }
 
+  object InitializeClassifiers {
```

Comment:
Can you add some documentation on what `initialization` means? Should we call this `PreExtract` instead?